pandas/tests/io/json/test_deprecated_kwargs.py

```diff
@@ -0,0 +1,34 @@
+"""
+Tests for the deprecated keyword arguments for `read_json`.
+"""
+
+import pandas as pd
+import pandas.util.testing as tm
+from pandas.util.testing import assert_frame_equal
+
+from pandas.io.json import read_json
+
+
+def test_deprecated_kwargs():
+    """
+    Check whether FutureWarning is produced if `orient`
+    argument passed as positional.
+    """
+    df = pd.DataFrame({"A": [2, 4, 6], "B": [3, 6, 9]}, index=[0, 1, 2])
+    with tm.assert_produces_warning(FutureWarning):
+        assert_frame_equal(df, read_json(df.to_json(orient="split"), "split"))
```

Comment:
Can you minimize the amount of code in the context manager? I think just `read_json(buf, "split")`.

Also, do we need to test all three of these? What differs between them?